lib/profile/save_profile/sections/education/education_item.dart

```diff
@@ -338,19 +309,119 @@ class _SaveEducationItemState extends State<SaveEducationItem> {
             buildDateRangeRow(school)
           ]
         )
       ),
     ];
   }
 
+  Widget buildCenterAno() {
+    return Container(
+      padding: EdgeInsets.only(bottom: 60.0),
+      width: screenWidth,
+      child: Column(
+        children: <Widget>[
+          Image.asset(
+            'images/two_anos_talking.png',
+            height: screenHeight * 0.20,
+          ),
+          Container(
+            height: 1.5,
+            color: Colors.black
+          )
+        ]
+      )
+    );
+  }
+
+
+  Widget buildSchoolNameField(School school) {
+    return Container(
+      padding: EdgeInsets.only(bottom: 20.0, left: 60.0, right: 60.0),
+      child: FormBuilderTextField(
+        attribute: 'school',
+        initialValue: school.name,
+        focusNode: schoolFocus,
+        maxLines: 1,
+        style: fieldTextStyle(color: Theme.of(context).primaryColor),
+        onChanged: (value) => store.dispatch(
+        UpdateSaveEducationState(
+          school.copyWith(
+            name: value,
+            endYear: school.endYear
+          )
+        )
+      ),
+        decoration: fieldDecoration(
+          context,
+          isFocused: isSchoolNameFocused,
+          isInvalid: isSchoolNameInvalid,
+          hintText: "Enter your institution",
+          icon: AspireIcons.lock
+        ),
+        validators: [
+          FormBuilderValidators.required(),
+          FormBuilderValidators.maxLength(100),
+        ],
+        keyboardType: TextInputType.text,
+      )
+    );
+  }
+
+
+  Widget buildDegreeField(School school) {
+    return Container(
+      padding: EdgeInsets.only(bottom: 20.0, left: 60.0, right: 60.0),
+      child: FormBuilderTextField(
+        attribute: 'degree',
+        initialValue: school.program,
+        focusNode: degreeFocus,
+        maxLines: 1,
+        style: fieldTextStyle(color: Theme.of(context).primaryColor),
+        onChanged: (value) => store.dispatch(
+        UpdateSaveEducationState(
+          school.copyWith(
+            program: value,
+            endYear: school.endYear
+          )
+        )
+      ),
+        decoration: fieldDecoration(
+          context,
+          isFocused: isDegreeFocused,
+          isInvalid: isDegreeInvalid,
+          hintText: "Enter your credential",
+          icon: AspireIcons.lock
+        ),
+        validators: [
+          FormBuilderValidators.required(),
+          FormBuilderValidators.maxLength(100),
+        ],
+        keyboardType: TextInputType.text,
+      )
+    );
+  }
+
+  Widget buildAddButton() {
+    return Container(
+      padding: EdgeInsets.only(top:100.0, bottom: 20.0, 
+      left: 60.0, right: 60.0),
+      child: PrimaryButton(
+        isLight: true,
+        text: widget.editMode ? 'Edit' : 'Add',
+        onPressed: () {
+          if (_saveEducationItemKey.currentState.saveAndValidate()) {
+            handleSaveEducation();
+          } else {
+            print("Validation failed.");
```

Comment:
Merge all these line into the `handleSaveEducation` function

Reply:
I don't understand this comment.

Reply:
Move the conditional to the `handleSaveEducation()`! The `onPressed` is too long and we can move the `if` and `else` in `handleSaveEducation`.

Reply:
Your `handleSaveEducation()` function should look like:
```
void handleSaveEducation() {
    if (_saveEducationItemKey.currentState.saveAndValidate()) {
        store.dispatch(SaveSchool(payload: widget.editMode));
        setState(() { isActive = false; });
        unfocusFields();
        Navigator.pop(context);
   } else {
       print("Validation failed.");
  }
```

Then you could just do this instead under the Add/Edit button widget:
```
onPressed: handleSaveEducation
```

Reply:
That's make sense!